packages/sandcastle/public/gallery/billboards/sandcastle.yaml

```diff
@@ -0,0 +1,10 @@
+id: billboards
+legacy-id: Billboards.html
+title: Billboards
+thumbnail: ./Billboards.jpg
```

Comment:
Adding the `thumbnail` property here lets it be named anything. We had discussed maybe not including this and consistently requiring a `thumbnail.png` or something the same as I am for `index.html` and `main.js`. Not sure which approach is better here.
The value is optional regardless if we check for a file or it's defined here.